Rethink the process

szepeviktor opened the pull request:
@Starttoaster There may be errors in this!

szepeviktor committed:
Rethink the process

Starttoaster commented:
Hey, @szepeviktor ! Great work, and I have been considering doing something similar to this. I will be making some time to test this out before merging but wanted to say thank you for your interest and support in advance!

Starttoaster committed:
Fixed directory permission discrepancy

/opt/traefik received the ownership of the current user, but /opt/wiki did not.

Starttoaster committed:
Minor text fix

Sorry, the no-space thing there bugged me a bit..

Starttoaster committed:
Minor text fix

Been meaning to change that optional DDNS paragraph a bit.

Starttoaster reviewed (approved):
Excellent work! I made some minor alterations based on things I noticed.
